packages/tradershub/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -69,19 +64,23 @@ const JurisdictionModal = () => {
                     <Modal.Header onRequestClose={closeModal}>
                         <Text weight='bold'>{jurisdictionTitle}</Text>
                     </Modal.Header>
-                ) : null}
+                ) : (
+                    <DynamicLeverageTitle />
+                )}
                 <Modal.Body
                     className={twMerge(
-                        'p-0 flex flex-col relative min-h-0 overflow-auto lg:w-auto lg:h-auto lg:p-8 lg:min-w-[1100px]'
+                        'p-0 flex flex-col relative min-h-0 overflow-auto w-screen h-screen lg:w-auto lg:h-auto lg:p-8 lg:min-w-[1100px]',
```

Comment:
@thisyahlen-deriv please sort this classes

Reply:
Headwind extension has some issues, sorting is not working at the moment for me